packages/core/bootstrap/src/lib/util.ts

```diff
@@ -470,3 +471,22 @@ export const buildUrl = (baseUrl: string, pathTemplate = '', params = {}, whitel
   new URL(buildUrlPath(pathTemplate, params, whitelist), baseUrl).toString()
 
 //  URL Encoding
```

Comment:
Is this comment relevant to the section below? Just looks like it didn't belong here

Reply:
Seems to be the "end" tag of a begin-end comment pair, I'd probably refactor it and extract it out into its own file rather than comment, but I'll leave that for another PR so as not to clutter this one